app/src/main/res/values/strings.xml

```diff
@@ -9,5 +9,9 @@
     <string name="recipe_picker_high_cat">Well, ain\'t you having fun. *winks*</string>
     <string name="recipe_picker_cool_cat">The Bad Boy of the Block. You can\'t resist him, huh?</string>
     <string name="recipe_picker_surprised_cat">Surprise surprise!</string>
+    <string name="recipe_picker_google_login">Google login</string>
 
+    <!-- Google login -->
+    <string name="google_logout">Logout</string>
```

Comment:
We should use the same prefix for every `string` in the recipe (the same as we do with features in an app). In this case it would be `google_login_logout`. I know that it looks weird but sticking to conventions makes it easier to find.